pkpdapp/pkpdapp/settings.py

```diff
@@ -147,7 +147,7 @@
 # Static files (CSS, JavaScript, Images)
 # https://docs.djangoproject.com/en/3.0/howto/static-files/
 STATIC_URL = '/static/'
-STATIC_ROOT = os.path.join(BASE_DIR, 'staticfiles')
+STATIC_ROOT = os.path.join(BASE_DIR, 'static')
```

Comment:
Do we have to change this to 'static'? It seems more conventional to name this 'staticfiles'

Reply:
it's just easier when writing the nginx config. nginx seems to want the actual directory to be the same name as the url. There might be a configuration option to change this, but I couldn't find it

Reply:
Ok I personally also don't know why the convention is to use staticfiles when the URL is called static.